pymc3/distributions/timeseries.py

```diff
@@ -439,7 +439,7 @@ def __init__(
 
         self.init = init
         self.innovArgs = (mu, cov, tau, chol, lower)
-        self.innov = multivariate.MvNormal.dist(*self.innovArgs)
+        self.innov = multivariate.MvNormal.dist(*self.innovArgs, shape=self.shape[-1])
```

Comment:
The idea here is to pass the spatial dimension to the `MvNormal` function. I think there might be a better way to do this.